views/rosterview/login.hbs

```diff
@@ -2,13 +2,22 @@
 
 <form action="/user/login" method="post">
     <div class="col s12">
-        <input type="email" name="email" class="text col s4 offset-m4" placeholder="Email">
-        <input type="password" name="password" class="text col s4 offset-m4" placeholder="Password">
-        <div class="col s3 offset-m6">
-            <button class="btn waves-effect waves-light col s7" type="submit">Login
-                <i class="material-icons right">login</i/>
-            </button>
-        </div>
+        <input type="email" name="email" class="text col s4 offset-m4" placeholder="Email" required>
```

Comment:
You are using the browser to post back to the server, you can just add the required flag and push validation on to the browser when submitting. No need then to validate on the server if this is a valid email format. Similar situation with the password, browser wont submit unless present with a valid email.